code/server/models/Goal.js

```diff
@@ -3,12 +3,12 @@ const Schema = mongoose.Schema;
 
 // Goal Schema
 const goalSchema = new Schema({
-  goalId: { type: Number, required: true },
-  type: { type: String, required: true },
-  targetValue: { type: Number, required: true },
-  unit: { type: String, required: true },
-  createdAt: { type: Date, default: Date.now },
-  userId: { type: Number, required: false }
```

Comment:
we should also update the two goals created in initdb.js to reflect this new schema